src/modules/app/route-configs/RouteConfigInterface.js

```diff
@@ -0,0 +1,43 @@
+// @flow
```

Comment:
There is no need to name this _Interface. Just RouteConfig is enough in my opinion.

https://softwareengineering.stackexchange.com/questions/117348/should-interface-names-begin-with-an-i-prefix